Tools/baselineDef.cc

```diff
@@ -315,16 +320,16 @@ void BaselineVessel::PassBaseline()
   if (tr->getVar<unsigned int>("run") > 312526) // Set weight to 1 for Data
```

Comment:
This hack can be removed as it now does nothing

Reply:
Can I remove this from all three places that it shows up?